client/stylesheets/controls.styl

```diff
@@ -151,8 +146,6 @@
       resize vertical
       table
         font-size 12px
-        background rgba(232, 255, 255, 0.75)
-        color black
```

Comment:
With opaque white background (the intended background from Bootstrap) I think `#333` looks good.